cwms-radar-client/src/test/resources/radar/json/location.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
not related to this change

Reply:
Once PR6 is merged, i can do another rebase off main, which will remove this